app/controllers/api/base_controller.rb

```diff
@@ -59,6 +59,22 @@ def index
       render_collection(@req.subject, res, opts)
     end
 
+    def create
+      if @req.resources.any? { |r| parse_id(r, @req.subject.to_sym) }
+        raise BadRequestError, "Resource id or href should not be specified for creating a new #{@req.subject}"
+      end
```

Comment:
You should leave this for a followup PR. There's a whole can of worms to fix here to try and stop such errors (we should really be doing a bulk update in a transaction to rollback on *any* failures, not just specifying an ID when you shouldn't)